From 6b53f7b003f429b0c3958c07304529e9a6a78ead Mon Sep 17 00:00:00 2001
From: Olivier Brunel <jjk@jjacky.com>
Date: Sat, 14 Jan 2017 18:17:01 +0100
Subject: [PATCH 5/8] prefs_filtering: Add missing shadow on the list

Looks better. Also more consistent, since such shadow is used on about
every other window (conditions, actions...)

Signed-off-by: Olivier Brunel <jjk@jjacky.com>
---
 src/prefs_filtering.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/prefs_filtering.c b/src/prefs_filtering.c
index ea2d195a1..3a57c24a9 100644
--- a/src/prefs_filtering.c
+++ b/src/prefs_filtering.c
@@ -508,6 +508,8 @@ static void prefs_filtering_create(void)
 
 	cond_scrolledwin = gtk_scrolled_window_new (NULL, NULL);
 	gtk_widget_show (cond_scrolledwin);
+	gtk_scrolled_window_set_shadow_type(GTK_SCROLLED_WINDOW(cond_scrolledwin),
+					GTK_SHADOW_ETCHED_IN);
 	gtk_widget_set_size_request (cond_scrolledwin, -1, 150);
 	gtk_box_pack_start (GTK_BOX (cond_hbox), cond_scrolledwin,
 			    TRUE, TRUE, 0);
-- 
2.11.1

